### ISSUE TITLE
Inconsistent tooltip for Decode block

### ISSUE THREAD

**Issue by jnoche** [mortbopet/Ripes#175]
I'm using version 2.2.3 with the 5-stage RISC-V processor without forwarding or hazard detection.  My program starts with the instruction addi x10 x0 2.  I clock the circuit and reach the part where this instruction is in stage 2.  I hover the mouse cursor over the 'opcode' signal output wire of the 'Decode' block and a tooltip appears showing the text "opcode: ADDI", which is what is expected.
<img width="914" alt="ADDI" src="https://user-images.githubusercontent.com/63948655/142749038-fd998c33-1cc9-43ca-a857-60648e910b1d.png">

But when I move the mouse over the 'Decode' block, the tooltip that appears shows the text "opcode: NOP", which is not what is expected.
<img width="914" alt="NOP" src="https://user-images.githubusercontent.com/63948655/142749052-85c62627-6700-4b1c-b254-4e4429f048e0.png">

**Comment by jnoche** [comment]
Oops.  I forgot to mention that this is for the 32-bit processor.

After I posted my comment above, I tried to see if the error pops up in the other 32-bit processor types.  It seems that it doesn't.  Then when I tried to see if the error pops up for the 5-stage processor without forwarding or hazard detection, it seems that the problem disappeared.  Now, hovering the mouse over the signal shows the correct opcode, and hovering it over the block doesn't show any tooltip.  So I'm currently unable to replicate the error.
